addons/io_scene_gltf2/blender/exp/gltf2_blender_gather.py

```diff
@@ -58,3 +59,9 @@ def __gather_animations(blender_scene, export_settings):
     for blender_object in blender_scene.objects:
         animations += gltf2_blender_gather_animations.gather_animations(blender_object, export_settings)
     return animations
+
+
+def __gather_extras(blender_object, export_settings):
+    if export_settings['gltf_extras']:
```

Comment:
Please don't use a literal string here.
```suggestion
    if export_settings[gltf2_blender_export_keys.EXTRAS]:
```